internal/rpcapi/plugin.go

```diff
@@ -91,3 +96,19 @@ func serverHandshake(s *grpc.Server, opts *serviceOpts) func(context.Context, *t
 type serviceOpts struct {
 	experimentsAllowed bool
 }
+
+func newServiceDisco(cliConfig *terraform1.Config) (*disco.Disco, error) {
```

Comment:
I suggest renaming this variable to just `config` now, to make it match everything else!